### DISCUSSION TITLE
diary/2018/10/26/gap-year-and-socar/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#103] [Comments]
#  Gap Year 및 쏘카 이직 이야기

https://zzsza.github.io/diary/2018/10/26/gap-year-and-socar/

**Comment by zzsza** [comment]
<p>퇴사후에 공부를 더 많이 하신것 같네 ㅋㅋㅋ</p>

>_This comment was made by Disqus. 2018-10-26 09:08:05 장지수_

**Comment by zzsza** [comment]
<p>ㅋㅋㅋㅋㅋㅋ퇴사후에 원없이 공부했지만 더 하고 싶었다..!</p>

>_This comment was made by Disqus. 2018-10-27 09:11:18 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>감사히 읽었습니다.<br>존경스럽습니다. :)</p>

>_This comment was made by Disqus. 2018-10-29 10:43:15 Hyun Seok Jeong_

**Comment by zzsza** [comment]
<p>감사합니다!! :)<br>더 열심히 해보겠습니다!_!</p>

>_This comment was made by Disqus. 2018-10-31 09:30:12 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>블로그 잘읽었습니다. 혹시 회사 탐방은 대표님께 어떤식으로 첫 멘트를 건내야 할까요?</p>

>_This comment was made by Disqus. 2018-12-14 03:27:35 이경주_

**Comment by zzsza** [comment]
<p>안녕하세요! 요새 disqus 알람이 잘 안오네요;; <br>저는 보통 회사에 대한 관심을 표현하고, 그 회사의 기술에 대해 호기심을 갖는 편입니다! <br><a href="https://eiec.kdi.re.kr/publish/nara/year/view.jsp?idx=11625" rel="nofollow noopener" title="https://eiec.kdi.re.kr/publish/nara/year/view.jsp?idx=11625">https://eiec.kdi.re.kr/publ...</a></p><p>이 스타트업에 방문했는데, 전혀 안면이 없었지만 대표님에게 페이스북 친구 요청을 했습니다<br>친구 요청을 수락해주셔서, 메세지로 저에 대한 간단한 설명을 드렸고, 혹시 나중에 시간되면 회사에 방문해보고 싶다고 말했어요-! (태아 사진을 어떻게 GAN으로 만들까.. 이런 고민도 하고!)</p><p>그러면 거의 대부분 "네 오세요~" 하시는데 거의 바로 약속을 잡곤 했어요 :)</p>

>_This comment was made by Disqus. 2018-12-16 11:21:22 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>중간에 "바닥부터시작하는..." 발표자료 인상깊게봤어요. 멋지십니다</p>

>_This comment was made by Disqus. 2018-12-31 10:15:01 Jaeyoung Park_

**Comment by zzsza** [comment]
<p>you can post with english ?</p>

>_This comment was made by Disqus. 2020-02-02 05:58:59 dicnet_

**Comment by uuuuooii** [comment]
안녕하세요! 혹시 마이크로 소프트웨어 잡지 구독은 어떻게 하셨는지 알려주실 수 있나요 저도 관심이 있어서요!  ㅎ

**Comment by KHG0217** [comment]
글 정말 잘읽었습니다. 데이터 수집 및 데이터관리 업무로 2년정도 경력을 쌓다가 현재 이직을 준비하는 과정에서 우연히 검색하다 알게되었는데, 많이 참고하고 공부하겠습니다 :) 데이터엔지니어, 데이터사이언티스트 진로계획에 많이 도움이될것같습니다 !
